src/Fixtures/en/frisbeeDrills.en.js

```diff
@@ -6573,6 +6577,104 @@ const fish = {
   ],
 };
 
+const trident = {
+  id: 1050,
+  type: DrillTypes.FRISBEE,
+  author: 'Monkey',
+  title: 'The Trident',
+  image: 'https://zupimages.net/up/20/47/6ksu.jpg',
+  description: 'Players practice quickly identifying a target to pass to as quickly as possible.',
+  minimalPlayersNumber: 5,
+  inGame: 'Identifying the best passing option is paramount in a game.',
+  equipmentLabel: EquipmentLabels.BASIC,
+  equipment: '2 discs for 5 players',
+  durationInMinutes: 10,
+  intensity: Intensities.MODERATE,
+  goals: [FrisbeeGoals.MARK, FrisbeeGoals.THROWING],
+  seasonTiming: SeasonTimings.ANYTIME,
+  level: Levels.BEGINNER,
+  steps: [
+    {
+      id: 1,
+      title: 'The Trident',
+      animation: tridentAnimation,
+      repetition: '',
+      rest: '',
+      instruction:
+        'The thrower is looking to play with his teammate. He is marked (forehand or backhand) by the defender. He tries to make as many passes as possible in 1 minute.\nGoal for beginner: To play on the easiest to reach.\nExperienced goal: Try to break systematically.',
+    },
+  ],
+};
+
+const firstWall = {
+  id: 1051,
+  type: DrillTypes.FRISBEE,
+  author: 'Monkey',
+  title: 'The Wall',
+  image: 'https://zupimages.net/up/20/47/qhge.jpg',
+  description:
+    "An exercise to train at the first wall of a zone defense. The wall of a zone allows to slow down the opponent's offense.",
+  minimalPlayersNumber: 9,
+  inGame: "When a zone is set up, most teams put up a first wall to limit the opponents' attack zones.",
+  equipmentLabel: EquipmentLabels.BASIC,
+  equipment: '1 disc - 4 cones',
+  durationInMinutes: 15,
+  intensity: Intensities.MODERATE,
+  goals: [FrisbeeGoals.HANDLING, FrisbeeGoals.DEFENSE],
+  seasonTiming: SeasonTimings.ANYTIME,
+  level: Levels.INTERMEDIATE,
+  steps: [
+    {
+      id: 1,
+      title: 'The Wall',
+      animation: firtWallAnimation,
+      repetition: '',
+      rest: '',
+      instruction:
+        'The objective is to pass through the 1st wall of zone. The count is 10 per side. Throws over are forbidden.',
+    },
+  ],
+};
+
+const bull = {
```

Comment:
I have an error when I try to open this drill in the app

![Capture du 2020-11-19 18-38-30](https://user-images.githubusercontent.com/3810795/99702711-76b58280-2a96-11eb-8496-c75dcbebd6cf.png)